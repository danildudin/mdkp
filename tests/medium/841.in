30 9 0
939 231 668 957 868 90 934 569 210 617 477 961 949 554 373 453 53 193 643 146 673 824 277 173 592 990 67 847 862 13 
272 827 664 501 547 162 270 485 959 136 649 738 715 357 478 986 241 731 642 442 619 71 782 680 162 601 403 261 679 366 
407 608 985 34 55 966 517 1000 515 433 170 807 325 612 633 471 494 256 606 881 157 29 953 641 724 234 894 67 685 260 
8 398 61 619 603 99 169 319 922 330 526 978 794 775 478 514 640 919 268 957 284 413 118 373 711 303 734 182 869 320 
270 549 312 797 117 466 163 157 407 145 522 355 694 202 813 981 938 955 835 736 534 640 872 643 596 600 588 249 703 442 
67 670 15 134 704 928 442 445 418 728 339 977 643 495 325 664 398 335 588 264 835 870 670 719 229 864 938 470 977 296 
765 436 427 450 367 147 587 906 525 155 64 492 120 750 460 763 40 172 181 704 92 38 124 469 21 716 408 380 417 225 
289 570 276 617 174 956 323 59 427 590 232 704 920 908 169 296 166 777 269 853 86 837 444 269 419 263 685 299 259 388 
90 154 558 365 190 79 384 766 614 192 953 304 328 786 239 181 654 301 203 67 892 194 611 304 967 351 634 449 427 317 
339 946 47 877 779 150 556 812 739 506 771 489 343 229 528 827 156 184 823 479 547 862 730 527 21 362 571 141 615 303 
7538 5709 6755 9223 7085 5814 6980 6351 5812 
