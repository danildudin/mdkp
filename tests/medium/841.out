10945
13
0 2 3 4 6 11 12 13 21 24 25 27 28 
